src/_actions/incidenceReports.actions.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+import constants from '../_constants';
+
+// constants
+const {
+  LOAD_INCIDENCE_REPORTS_SUCCESS,
+  LOAD_INCIDENCE_REPORTS_FAILURE,
+  LOAD_INCIDENCE_REPORTS_START,
+} = constants;
+
+/**
+ * Load incidence reports success
+ *
+ * @param {array} incidenceReports
+ * @return {object} type and payload
+ */
+export const loadIncidenceReportsSuccess = incidenceReports => (
+  { type: LOAD_INCIDENCE_REPORTS_SUCCESS, incidenceReports }
+);
+
+/**
+ * Incidence reports thunk
+ *
+ * @return {(dispatch:any)=>Promise<TResult2|TResult1>}
+ */
+export const loadIncidenceReports = (page = 1, limit = 10) =>
+  (dispatch) => {
+    dispatch({ type: LOAD_INCIDENCE_REPORTS_START });
+    return axios.get('http://art-backend.herokuapp.com/api/v1/incidence-reports/')
```

Comment:
This should be `axios.get('incidence-reports')`. The API URL is supposed to be declared in your `.env` as `REACT_APP_API_URL`. Read [this slack message](https://andela.slack.com/archives/GAMEXMPFV/p1528813884000969)